### PR TITLE
Use resource-jsonld instead of wikibase-entity

### PR DESCRIPTION
**Pull request by Tpt**
Note: depends on https://github.com/ProjetPP/PPP-WebUI/pull/36
Should be deployed by @Tpt

### CODE REVIEW

**Comment by ProgVal** [comment]
“Should be deployed by @Tpt” -> why?

**Comment by Tpt** [comment]
Because I'm not sure that all will work well so I want to be there to check if everything is good.

**Comment by ProgVal** [comment]
:|

**Comment by ProgVal** [comment]
You should update the tests in the integration repo, by the way

File: composer.json [hunk 1]
```diff
@@ -23,9 +23,6 @@
 		"ppp/wikidataquery-api": "~1.1",
 		"ppp/libmodule": "*"
 	},
-	"require-dev": {
-		"whatthejeff/nyancat-phpunit-resultprinter": "~1.2"
-	},
```
Comment (ProgVal) [inline composer.json]: :+1:

**Pull request merged**